spec/ewb/extensions/IEC61970/Base/Wires/TransformerCoolingType.yaml

```diff
@@ -0,0 +1,26 @@
+name: TransformerCoolingType
```

Comment:
I also forgot to commit this or I have the wrong data all together... the diff is a little weird.